resources/build/build-utils.sh

```diff
@@ -48,6 +48,18 @@ function die () {
     exit 1
 }
 
+# Used to build an identifier usable to prefix builder_report outputs to more clearly identify
+# the calling script.
+#
+# Assumes that `findRepositoryRoot` has already been called.
+function buildScriptIdentifier() {
```

Comment:
Only if you're fine with moving `THIS_SCRIPT_PATH` below the call to `findRepositoryRoot`, which is notably lower in `build_utils.sh` - line 195 or so, rather than line 38.

Reply:
Ah yes, I see. OK, leave it here is fine, just add a comment up above to note that THIS_SCRIPT_IDENTIFIER is set further down. (Unless THIS_SCRIPT is not used earlier, in which case put them all in here?)

Reply:
> (Unless THIS_SCRIPT is not used earlier, in which case put them all in here?)

It isn't, so I'll do that.